src/lib/radon/index.ts

```diff
@@ -17,110 +13,125 @@ export {
     RadonInteger,
     RadonString,
     RadonMap,
-    RadonScript as RadonScriptWrapper
+    RadonScript as RadonScriptWrapper,
+    RadonEncodings,
 } from './types'
 
-export class RadonSLA {
-    public readonly numWitnesses: number;
-    public readonly unitaryFee: number;
-    constructor (numWitnesses: number, unitaryFee: number) {
-        this.numWitnesses = numWitnesses
-        this.unitaryFee = unitaryFee
-    }
+export { HexString as RadonBytecode } from "../types"
+
+import { RadonAny, RadonString, RadonOperator } from './types'
+export function RadonScript<InputType extends RadonAny = RadonString>(inputType: { new (ops?: RadonOperator): InputType; }): InputType {
+    if (!inputType) throw TypeError("An InputType must be specified when declaring a new Radon script") 
+    return new inputType();
 }
 
+// export type RadonSLA = {
```

Comment:
Should we remove the commented code?

Reply:
Yep. Ultimately superseded by DataRequestParams in the "crypto" module.